### ISSUE TITLE
take a look for analysis and issues report for improve your code

### ISSUE THREAD

**Issue by johnfelipe** [duracellko/planningpoker4azure#128]
[2023-07-07-planningpoker4azure-9.9-analysis-report.docx](https://github.com/duracellko/planningpoker4azure/files/11983824/2023-07-07-planningpoker4azure-9.9-analysis-report.docx)
[2023-07-07-planningpoker4azure-9.9-issues-report.xlsx](https://github.com/duracellko/planningpoker4azure/files/11983825/2023-07-07-planningpoker4azure-9.9-issues-report.xlsx)

only suggestions for go to 0 in bugs, vulnerabilities and more...

**Comment by duracellko** [comment]
Thank you very much. I will have a look.

**Comment by duracellko** [comment]
Sonar analyzer was added to the solution to have continuous checks for the issues.

The only unresolved issue is with the CDN integrity. However, when testing the resolution browser requires correct CORS headers at CDN to check the integrity. And it seems the required CORS header is missing. Therefore I decided to ignore this issue. It has low priority according to Sonar, and resolving it would require significant effort.
